modules/cpmso-yaml-provider-spec-nutanix.adoc

```diff
@@ -78,21 +78,39 @@ You must use the `Legacy` boot type in {product-title} {product-version}.
 ====
 Clusters that use {product-title} version 4.15 or later can use failure domain configurations.
 
-If the cluster is configured to use a failure domain, this parameter is configured in the failure domain.
+If the cluster uses a failure domain, configure this parameter in the failure domain.
 If you specify this value in the provider specification when using failure domains, the Control Plane Machine Set Operator ignores it.
 ====
 <4> Specifies the secret name for the cluster. Do not change this value.
 <5> Specifies the image that was used to create the disk.
 <6> Specifies the cloud provider platform type. Do not change this value.
 <7> Specifies the memory allocated for the control plane machines.
 <8> Specifies the Nutanix project that you use for your cluster. In this example, the project type is `name`, so there is a `name` stanza.
-<9> Specifies a subnet configuration. In this example, the subnet type is `uuid`, so there is a `uuid` stanza.
+<9> Specify one or more Prism Element subnet objects.
+In this example, the subnet type is `uuid`, so there is a `uuid` stanza.
+A maximum of 32 subnets for each Prism Element failure domain in the cluster is supported.
++
+[IMPORTANT]
+====
+The following known issues with configuring multiple subnets exist in {product-title} version 4.18:
+
+* Adding subnets above the existing subnet in the `subnets` stanza causes a control plane node to become stuck in the `SchedulingDisabled` state.
```

Comment:
@jeana-redhat How about changing "causes a control plane node to become stuck in the `SchedulingDisabled` state" to "causes a control plane machine to become stuck in the `Deleting` state", because sometimes I see one control plane machine stuck in `Deleting` but all nodes shows `Ready` status.